### PR TITLE
Update for Issue #47

### PR DESCRIPTION
**Pull request by MiguelOliveira1200874**
**Pull Request: Update for Issue #47**

**Description:**
This pull request addresses Issue #47 by updating the README.md file to provide a more comprehensive and up-to-date overview of the Antirecurso project. The changes made include:

1. Updated the introduction section to provide a clearer and more concise description of the project.
2. Reorganized and expanded the "About the Project" section to provide a better understanding of the evolution of Antirecurso from a mobile app to a comprehensive study platform.
3. Added a new section, "Usage Examples," to provide step-by-step instructions on how to use Antirecurso and included relevant screenshots for better visualization.
4. Updated the "Tech Stack" section to reflect the transition to the Next.js framework and provided a brief explanation of Next.js, Tailwind CSS, and Alpine.js.
5. Updated the "Contributions" section to provide clearer instructions for contributing to the project.
6. Added a new section, "License Information," to provide information about the project's open-source license and the freedoms it provides.
7. Made minor formatting and styling improvements throughout the README.md file.

**Changes Made:**

- Replaced outdated information with updated project details.
- Added step-by-step usage examples with relevant screenshots.
- Updated the tech stack section to reflect the transition to Next.js.
- Added a placeholder table for release updates.
- Clarified and expanded the contribution guidelines.
- Added a new section for license information.
- Made minor formatting and styling improvements.

Please review and let me know if any further changes are required. Thank you!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,64 +4,85 @@
...
 
-Whether you're using a computer, a tablet, or even your phone, you can log in to "antirecurso" and access a ton of exercises to help you prepare for your exams. These exercises are presented in a form of short quizzes with 10 random questions each. After answering them, the user gets a feedback with how many they got wrong and it goes into a scoreboard.
+Whether using a computer, tablet, or mobile phone, "antirecurso" provides numerous exercises in the form of short quizzes, each with 10 random questions. Upon completion, users receive immediate feedback and their performance is tracked on a scoreboard.
 
-So, if you're a student looking for a little extra help with your studies, definitely check out "antirecurso" <a href="https://antirecurso.nei-isep.org" target="_blank">here</a>. It's a game-changer when it comes to exam prep!
+If you're a student seeking extra academic resources, check out "antirecurso" <a href="https://antirecurso.nei-isep.org" target="_blank">here</a>. It's a comprehensive tool for exam preparation.
```
Comment (ricardo177m) [inline README.md:21]: Change all references of "antirecurso" to Antirecurso (no quotes & uppercase).

File: README.md [hunk 2]
```diff
@@ -4,64 +4,85 @@
...
+
+For this project, we've transitioned to using the Next.js framework. It brings productivity and agility in development and allows us to build server-side rendering and static web applications.
+
+Here's a quick rundown of the tech stack:
+
+- **Next.js**: A top-tier open-source React front-end development web framework that enables functionality such as server-side rendering and generating static websites for React based web applications.
+
+- **Tailwind CSS**: A utility-first CSS framework packed with classes like flex, pt-4, text-center and rotate-90 that can be composed to build any design, directly in your markup.
+
+- **Alpine.js**: A rugged, minimal framework for composing JavaScript behavior in your markup. It’s like Tailwind for JavaScript.
```
Comment (ricardo177m) [inline README.md:58]: We do not use Alpine.js anymore; you could instead add a reference to Framer Motion, for example.

**Review by ricardo177m — CHANGES_REQUESTED** [review]

**Comment by MiguelOliveira1200874** [comment]
@ricardo177m Corrected everything you said and added a RoadMapSection according to some of your instructions https://github.com/Nucleo-Estudantes-Informatica-ISEP/antirecurso/pull/53/commits/9766bd0b6d8b98ae3d5f8538d8fad550cd186da0

**Comment by ricardo177m** [comment]
@MiguelOliveira1200874 Well done! One more thing: we're missing the fork part from the contributions section. Could you please add it and fix a typo in "Contribuitions"?

**Comment by MiguelOliveira1200874** [comment]
@ricardo177m Added the Contact Us Section and corrected what you asked for.

PS : Please check the email i sent you on [support.antirecurso@nei-isep.org](mailto:support.antirecurso@nei-isep.org).

**Comment by ricardo177m** [comment]
LGTM!

**Pull request merged**